curriculum/challenges/english/25-front-end-development/quiz-css-variables/66ed9018f45ce3ece4053eb9.md

```diff
@@ -17,439 +17,445 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+If you’ve declared a custom property for animation speed using `@property`, and a user adjusts their settings for animations in their browser, how would that affect your custom property? Would they be able to see the changes, or does it stick to the value you’ve set?
 
 #### --distractors--
 
-Placeholder distractor 1
+The animations will run slower regardless of user settings
 
 ---
 
-Placeholder distractor 2
+The user’s settings override your defined speed
 
 ---
 
-Placeholder distractor 3
+The animations will completely stop if the user disables them
 
 #### --answer--
 
-Placeholder answer
+The user’s preferences will take priority, and they can adjust how the animations play out
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In the following code, which value will color have if `--main-color` is not defined?
+
+```css
+p {
+  color: var(--main-color, pink);
+}
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+Undefined
 
 ---
 
-Placeholder distractor 2
+No color will be applied
 
 ---
 
-Placeholder distractor 3
+main-color
 
 #### --answer--
 
-Placeholder answer
+pink
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a valid way to declare a custom property in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+`--background-color: #333;`
 
 ---
 
-Placeholder distractor 2
+`custom-variable: #333;`
 
 ---
 
-Placeholder distractor 3
+`var(--background-color: #333);`
 
 #### --answer--
 
-Placeholder answer
+`$background-color: #333;`
```

Comment:
This doesn't seem quite right to me. https://developer.mozilla.org/en-US/docs/Web/CSS/--* 

Mind elaborating?

Reply:
yea i got your point..correcting the answer now it should be `--background-color: #333;`